.github/workflows/main.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
```

Comment:
be nice to run on a schedule too https://github.com/kturney/ember-mapbox-gl/blob/master/.github/workflows/ci.yml#L10